docs/projects/README.md

```diff
@@ -0,0 +1,470 @@
+
+## Where documents are kept, how they are named, and how they are reviewed
+
+For each project, planning documents should be kept in the `rfcs/` directory of
+this repository. Each project should have its own subdirectory. Projects
+predating this document have been updated to follow this structure. Individual
+documents should be date-stamped in the `YYYYMMDD` format. An example of this
+entire structure follows:
+
+```
+| rfcs/
+   | service_metrics/
+      - YYYYMMDD-project_proposal.md
+      - YYYYMMDD-implementation_plan.md
+   | 3d_model_support/
+      - YYYYMMDD-project_proposal.md
+      - YYYYMMDD-implementation_plan_(catalogue).md
+      - YYYYMMDD-implementation_plan_(api).md
+      - YYYYMMDD-implementation_plan_(frontend).md
+```
+
+Using subdirectories makes it slightly easier to navigate an ever-growing list
+of planning documents. This is especially the case in projects similar to the 3D
+Models project in the example above that may require multiple implementation
+planning documents for each part of the application. In the process of planning
+or implementing some projects, we may develop additional supporting documents or
+appendixes that go beyond the required scope of the project planning process
+described in this document. Those documents should accompany the required ones
+in the same subdirectory for the project.
+
+All project documentation can be authored with any tool, but must be written in
+[GitHub Flavored Markdown](https://github.github.com/gfm/) including the
+available
+[GitHub-specific advanced formatting features](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting).
+
+Additionally, all documents should be formatted to pass the Prettier linting
+step required by the repository for Markdown documents.
+
+When submitting a new document for a project, the author should open a pull
+request with the document. The review process of the document will happen as a
+PR review. Once the document is accepted and approved, it is merged and the
+relevant step of the process is complete.
+
+## Providing project updates
+
+At all stages of a project (except for `Not Started` projects), weekly public
+updates are the best way for Openverse contributors and maintainers to
+understand the health of a project. Importantly, these updates allow us to
+allocate more time or material resources to projects that need them before work
+stalls completely.
+
+Project updates should be left as comments in the project threads. Even if the
+update is "the project is blocked." or "No developments this week.", the updates
+should still be posted. This removes ambiguity for folks visiting the project,
+avoiding questions like "I don't see an update this week; is it being worked on
+or not?"
+
+These updates are useful for folks checking in on a project but also for the
+project lead. It's an opportunity to _reflect on_ the status of the project and
+any blockers, or successes to be shared.
```

Comment:
Currently, our internal guideline is to leave project updates at least **once every two weeks**. I prefer this cadence as it reduces overhead work for the project lead, and that will remove the need to leave a sort of hollow notice of "No developments this week." as it's most probably that something has occurred in the last two weeks of the project. Note that we have already been struggling to maintain this cadence, so I'm worried about doing it even more often.

Reply:
@krysal this was a mistake, I agree with the two-week cadence. Will fix!